dev/tools/localization/gen_l10n.dart

```diff
@@ -0,0 +1,318 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/args.dart' as argslib;
+import 'package:path/path.dart' as path;
+
+import 'localizations_utils.dart';
+
+const String defaultFileTemplate = '''
+import 'dart:async';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:intl/intl.dart';
+
+import 'messages_all.dart';
+
+/// Callers can lookup localized strings with an instance of @className returned
+/// by `@className.of(context)`.
+///
+/// Applications need to include `@className.delegate()` in their app\'s
+/// localizationDelegates list, and the locales they support in the app\'s
+/// supportedLocales list. For example:
+///
+/// ```
+/// return MaterialApp(
```

Comment:
Maybe show the import here as well (assuming that the app is defined in lib/).

Reply:
Done